bundles/plugins/org.bonitasoft.studio.common.repository/src/org/bonitasoft/studio/common/repository/Repository.java

```diff
@@ -701,16 +698,11 @@ public void migrate(final IProgressMonitor monitor) throws CoreException, Migrat
     }
 
     protected BonitaBPMProjectMigrationOperation newProjectMigrationOperation(final IProject project) {
-        return new BonitaBPMProjectMigrationOperation(project, this).
-                addNature("org.eclipse.xtext.ui.shared.xtextNature").
-                addNature("org.bonitasoft.studio.common.repository.bonitaNature").
-                addNature(JavaCore.NATURE_ID).
-                addNature("org.eclipse.pde.PluginNature").
-                addNature("org.eclipse.jdt.groovy.core.groovyNature").
-                addBuilder("org.eclipse.jdt.core.javabuilder").
-                addBuilder("org.eclipse.xtext.ui.shared.xtextBuilder").
-                addBuilder("org.eclipse.pde.ManifestBuilder").
-                addBuilder("org.eclipse.pde.SchemaBuilder");
+        return new BonitaBPMProjectMigrationOperation(project, this).addNature("org.eclipse.xtext.ui.shared.xtextNature")
+                .addNature("org.bonitasoft.studio.common.repository.bonitaNature").addNature(JavaCore.NATURE_ID).addNature("org.eclipse.pde.PluginNature")
+                .addNature("org.eclipse.jdt.groovy.core.groovyNature").addBuilder("org.eclipse.jdt.core.javabuilder")
```

Comment:
previous formatting war more readable (and don't we have some constants for al these values?)